app/src/main/java/com/epfl/drawyourpath/challenge/trophy/Trophy.kt

```diff
@@ -0,0 +1,31 @@
+package com.epfl.drawyourpath.challenge.trophy
+
+import com.epfl.drawyourpath.R
+import com.epfl.drawyourpath.utils.Utils
+import java.time.LocalDate
+
+data class Trophy(
+    val tournamentId: String,
+    val tournamentName: String,
+    val tournamentDescription: String,
+    val date: LocalDate,
+    val ranking: Int,
+    val drawable: Int = R.drawable.award,
+    val dateAsString: String = Utils.getDateAsString(date),
+) {
+
+    companion object {

```

Comment:
Maybe add a comment to explain in which situation the sample of trophy will be used (in test scenarios for example...)